sequelizeManager.js

```diff
@@ -0,0 +1,22 @@
+import Sequelize from 'sequelize';
+
+export default class sequelizeManager{
+    constructor(){
+        // nothing for now
+    }
+    initialize (usr, psw, db, prt, engine){
+      // create new sequelize object
+      this.connection = new Sequelize(db, usr, psw, {
+              dialect: engine,
+              port:    prt
+      });
+
+      console.log('making request');
```

Comment:
let's keep these `console.log`s out of the main code base from now on. you can keep them locally, just refrain from committing them